experimental/PlaneCurve/ParaPlaneCurves.jl

```diff
@@ -0,0 +1,152 @@
+export parametrization_plane_curve, adjoint_ideal, rational_point_conic,
+       parametrization_conic, map_to_rational_normal_curve,
+       rat_normal_curve_anticanonical_map, rat_normal_curve_It_Proj_Odd,
+       rat_normal_curve_It_Proj_Even
+
+
+################################################################################
+function _tosingular(C::ProjPlaneCurve{fmpq})
+    F = C.eq
+    T = parent(F)
+    Tx = singular_ring(T)
+    return Tx(F)
+end
+
+function _fromsingular_ring(R::Singular.PolyRing)
+    Kx = base_ring(R)
+    if typeof(Kx) == Singular.N_AlgExtField
+        FF, t = RationalFunctionField(QQ, "t")
+        f = numerator(FF(Kx.minpoly))
+        K, _ = NumberField(f, "i")
+    else
+        K = QQ
+    end
+    newring, _ = PolynomialRing(K, [string(x) for x in gens(R)])
+    return newring
+end
+
+function _tosingular_ideal(C::ProjCurve)
+    I = C.I
+    singular_assure(I)
+    return I.gens.S
+end
+
+@doc Markdown.doc"""
+    parametrization_plane_curve(C::ProjPlaneCurve{fmpq}, s::String = "local")
+
+Return an ideal `P` in a new ring `R` consisting of a rational parametrization
+of the rational plane curve `C`. The generators of `P` parametrize the three
+coordinates of the rational curve. The ground field of `R` is either QQ or some
+algebraic extension QQ(a). The optional string can be "normal", to compute the
+integral basis via normalization, or local (default), to make local analysis of
+singularities first and apply normalization separately.
+"""
+function parametrization_plane_curve(C::ProjPlaneCurve{fmpq}, s::String = "local")
+    F = _tosingular(C)
+    L = Singular.LibParaplanecurves.paraPlaneCurve(F, s)
+    R = L[1]
+    J = [L[2][i] for i in keys(L[2])][1]
+    S = _fromsingular_ring(R)
+    return ideal(S, J)
+end
```

Comment:
return ideal(S, J)
end
Commenting on lines +44 to +51

Please return vector here

Reply:
@delphinepol Returning vectors instead of ideals is a general remark: If the function returns a rational map, return vectors defining these. If the function returns an ideal, return an ideal.